### PR TITLE
Whats new for May

### PR DESCRIPTION
**Pull request by BillWagner**
Create the What's new page for May 2021.

There are two commits here. The first creates the new page, as we've done in the past.

The second is to trim the output to 3 months, and simplify the TOC. 

I looked at the traffic, and the 6 months old "what's new" gets ~10 visitors per month. The majority of the traffic is in the first month, then in drop by half for the 2nd, and even faster for the 3rd and later months.

I'm also hoping that I can create a GitHub action to open these PRs in the future, and the simpler format should make it easier to achieve that goal.

### CODE REVIEW

**Comment by BillWagner** [comment]
If you want me to keep the shorter cycle (3 months), up vote this comment. If you'd prefer leaving the 6 month window, downvote this comment.

File: docs/whats-new/2021-05.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: ".NET docs: What's new for May 2021"
+description: "What's new in the .NET docs for May 2021."
+ms.date: 06/01/2021
+---
+
+# .NET docs: What's new for May 2021
+
+Welcome to what's new in the .NET docs from May 1, 2021 through June 1, 2021. This article lists some of the major changes to docs during this period.
```
Comment (tdykstra) [inline docs/whats-new/2021-05.md:9]:
```suggestion
Welcome to what's new in the .NET docs from May 1, 2021 through May 31, 2021. This article lists some of the major changes to docs during this period.
```

File: docs/whats-new/2021-05.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+## .NET fundamentals
+
+### New articles
```
Comment (gewarren) [inline docs/whats-new/2021-05.md:13]: Remove descriptions in this section?

File: docs/whats-new/2021-05.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+- [NETSDK1141: Unable to resolve the .NET SDK version as specified in the global.json](../core/tools/sdk-errors/netsdk1141.md) - Create error page NETSDK1141
+- [Inspect managed stack traces (dotnet-stack)](../core/diagnostics/dotnet-stack.md) - Add dotnet-stack docs
+
+### Updated articles
+
+- [Language independence and language-independent components](../standard/language-independence.md) - Consolidate duplicate files
+
+## Azure SDK for .NET
+
+### New articles
```
Comment (gewarren) [inline docs/whats-new/2021-05.md:74]: Remove descriptions?

**Review by tdykstra — APPROVED** [review]
LGTM

**Review by gewarren — APPROVED** [review]

**Pull request merged**